tests/api/images.cpp

```diff
@@ -435,3 +435,42 @@ TEST_F(WithContext, Image1DBuffer) {
 
     auto image = CreateImage(CL_MEM_READ_WRITE, &format, &desc);
 }
+
+TEST_F(WithCommandQueue, ImageChannelGetter) {
+    const cl_channel_order order = CL_RGBA;
+    const cl_channel_type data_type = CL_SIGNED_INT32;
```

Comment:
Might be worth testing with a few different formats.

Reply:
Sure. I stayed with formats that should be available everywhere.